src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing.md

```diff
@@ -0,0 +1,65 @@
+
+A trace context generally consists of, at least, a `trace_id`, a `span_id`, and a `sampled` flag. The `trace_id` is a unique identifier for the trace, the `span_id` is a unique identifier for the request, and the `sampled` flag is a boolean that indicates whether the request should be traced. For instance, the [W3C Trace Context](https://www.w3.org/TR/trace-context/) defines the `Traceparent` header, whose value contains a `trace_id`, a `span_id` and a `sampled` flag, encoded as a dash (`-`) separated list of hexadecimal values. When a trace context header is attached to an HTTP request, we refer to it as being _propagated_ to the downstream service.
+
+## API
+
+| Class/Function                                                           | Description                                                                                                               |
+| :----------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [instrumentHTTP](/javascript-api/k6-experimental/tracing/instrumenthttp) | instruments the k6 http module with tracing capabilities.                                                                 |
+| [Client](/javascript-api/k6-experimental/tracing/client)                 | configurable Client that exposes instrumented HTTP operations and allows selectively instrumenting requests with tracing. |
+| [Options](/javascript-api/k6-experimental/tracing/options)               | Options to configure the tracing instrumentation behavior.                                                         |
+
+## Example
+
+This example demonstrates how to use the tracing API to instrument every HTTP request made in a script with tracing information.
+
+<CodeGroup labels={["example-tracing-module.js"]} lineNumbers={[]} showCopyButton={[true]}>
+
+```javascript
+import { check } from "k6";
+import tracing from "k6/experimental/tracing";
+import http from "k6/http";
+
+// instrumentHTTP will ensure that all requests made by the http module
+// from this point forward will have a trace context attached.
+//
+// The first argument is a configuration object that
+// can be used to configure the tracer.
+tracing.instrumentHTTP({
+  // possible values: "w3c", "jaeger"
+  propagator: "w3c",
+});
+
+export default () => {
+  // the instrumentHTTP call in the init context replaced
+  // the http module with a version that will automatically
+  // attach a trace context to every request.
+  //
+  // Because the instrumentHTTP call was configured with the
+  // w3c trace propagator, this request will as a result have
+  // a `traceparent` header attached.
+  const res = http.get("http://httpbin.org/get", {
+    headers: {
+      "X-Example-Header": "instrumented/get",
+    },
+  });
+
+  check(res, { "status is 200": (r) => r.status === 200 });
```

Comment:
Do we really need to use `check` in every example?

Also I am kind of against status checks given that [`expectedStatus`](https://k6.io/docs/javascript-api/k6-http/expectedstatuses/) does the same thing and the default already checks for 200-399 status checks

Reply:
We don't need to, removing them 👍